### PR TITLE
feat(file-uploader): Updated @uploadcare/blocks to @uploadcare/file-u…

### PR DESCRIPTION
**Pull request by egordidenko**
…ploader

## Checklist

- [ ] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: pyuploadcare/dj/conf.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ class SettingsType(typing_extensions.TypedDict):
     "use_legacy_widget": False,
     "use_hosted_assets": True,
     "widget": {
-        "version": "0.36.0",
+        "version": "1.4.0",
```
Comment (rsedykh) [inline pyuploadcare/dj/conf.py:70]: Maybe 1.* since no breaking changes?

**Review by dmitry-mukhin — CHANGES_REQUESTED** [review]
add changelog entry and we need new major pyuploadcare version.

**Comment by evgkirov** [comment]
Integration tests are failing.

**Comment by rsedykh** [comment]
Merge on behalf of @egordidenko

**Pull request merged**